rpc_util.go

```diff
@@ -778,6 +794,8 @@ func decompress(compressor encoding.Compressor, d []byte, maxReceiveMessageSize
 			// size is used as an estimate to size the buffer, but we
 			// will read more data if available.
 			// +MinRead so ReadFrom will not reallocate if size is correct.
+			//
+			// TODO: If we ensure that the buffer size is the same as the DecompressedSize, we can also utilize the recv buffer pool here.
```

Comment:
Super minor nit: please wrap this comment to 80 chars.

Reply:
Done. 4282fd5419fcaad11ceeb52586eabe98fd887e67